src/components/Home/Main/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import "../../../index"
+import { HorizontalCard } from "../common/horizontal-card"
+
+export function Main() {
+    return (
+        <main className="main-container">
+            {/* main banner */}
+            <section className="main-container top-section">
+                <div className="top-section-info">
+                    <h1>Limited Collection</h1>
+                    <h1>Big Discount</h1>
+                    <h3>Save 10% on your first purchase.</h3>
+                    <button>Shop Now</button>
+                </div>
+                <img className="top-section-image" src={require("./shoes.png")} alt="" />
+                <i className="fa fa-angle-left"></i>
+                <i className="fa fa-angle-right"></i>
+            </section>
+            {/* main feature-section */}
+            <section className="main-container featured-section">
+                <h1>Featured Categories</h1>
+                <div className="feature-section-info">
+                    <li className="feature-list">
+                        <img className="feature-img" src={require("./shoes.png")} alt="" />
+                        <span className="feature-badge">Nike Air Force 1 Low</span>
+                    </li>
+                    <li className="feature-list">
+                        <img className="feature-img" src={require("./shoes.png")} alt="" />
+                        <span className="feature-badge">Nike Air Max 270</span>
+                    </li>
+                    <li className="feature-list">
+                        <img className="feature-img" src={require("./shoes.png")} alt="" />
+                        <span className="feature-badge">Nike VaporMax</span>
+                    </li>
+                    <li className="feature-list">
+                        <img className="feature-img" src={require("./shoes.png")} alt="" />
```

Comment:
Try not import your images like this, instead put them in the public folder and then access it.

Reply:
Will do that thanks.

Reply:
Yeah. What @ivinayakg suggested is a better way to do it as you would need to add require and path everytime you render an image.